### PR TITLE
docs(guides): using the token modules

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: docs/pages/guides/adding-tokens.mdx [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+   Normally the only entity allowed to mint ERC-20 tokens is the namespace owner.
+   The easiest way to make sure `RewardSystem` has enough tokens to give is simply to give it enough tokens, and refill as needed.
+
+   ```solidity
+       // This is a workaround for issue
+       // https://github.com/latticexyz/mud/issues/2089
+       token.mint(address(worldAddress), 10 ** 6 * RewardSize.get());
+   ```
+
+   The issue is that even though `RewardSystem` is in a non-root namespace (`DDDD`), when called from `TaskSystem` it runs as the `World` address - so we need to give that address tokens to distribute.
```
Comment (alvrs) [inline docs/pages/guides/adding-tokens.mdx:528]: i'd prefer to not have this in the docs, let's wait for the issue to be fixed. Alternatively we can already merge the first part of the guide and hold back on the second part (controlling from a system). Or yet another alternative would be to use a root system for now so the behavior of transferring to the world is expected.

Comment (qbzzt) [reply docs/pages/guides/adding-tokens.mdx:528]: Any idea how long it will be until the bug is fixed? Do you want me to try and fix it?

Comment (alvrs) [reply docs/pages/guides/adding-tokens.mdx:528]: it's fixed now!

**Comment by alvrs** [comment]
Is this still stuck or ready for review?

**Comment by qbzzt** [comment]
Should be ready, but it has been so long, maybe I should go over it again
to make sure nothing changed.

**Comment by qbzzt** [comment]
Better to rewrite

**Pull request closed without merging**